snmp/datadog_checks/snmp/data/default_profiles/_generic-ospf.yaml

```diff
@@ -8,6 +8,7 @@ metrics:
         name: ospfNbrState
       - OID: 1.3.6.1.2.1.14.10.1.7
         name: ospfNbrEvents
+        metric_type: monotonic_count
```

Comment:
Note: this is possibly a "breaking change", but since the change is recent, I believe it shouldn't be a big issue.

Reply:
Yeah this was one of the changes suggested by the tooling and had the same feeling about it